src/metabase/util.cljc

```diff
@@ -841,9 +841,3 @@
   [xs]
   (or (empty? xs)
       (apply distinct? xs)))
-
-(defn deep-sort-map
```

Comment:
@bshepherdson are you okay with me removing it from utils and accompanied test? It's not used anywhere except serialization, and I'm replacing it with this PR anyway.

Reply:
Yeah, we don't need dead code lying around. Go ahead and remove it if no one else is using it.